CHANGELOG.md

```diff
@@ -4,7 +4,16 @@
 
 ## New features
 
-- [#2164: Enable cookie banner to set link styled as a button](https://github.com/alphagov/govuk-frontend/pull/2164)
+#### Allow users to use cookie banner buttons as links
+
+You can now add links styled as buttons to cookie banners.
+
+Use this feature only if you need to. For example, to display the **Hide** button because:
+
+- your cookie banner needs to work without JavaScript
+- the user has already accepted or rejected cookies - so, when they press **Hide** and the page reloads, the banner is hidden
+
+This was added in [pull request #2164: Enable cookie banner to set link styled as a button](https://github.com/alphagov/govuk-frontend/pull/2164)
```

Comment:
'Link' makes me think of navigation to another webpage. What, exactly, do we mean by 'link' in the context of the cookie banner?